edgexfoundry/edgex-go
# feat(metadata): implement PATCH device profile basic info API

**If your build fails**  due to your commit message not passing the build checks, please review the guidelines here: https://github.com/edgexfoundry/edgex-go/blob/main/.github/Contributing.md

## PR Checklist
Please check if your PR fulfills the following requirements:

- [ ] I have opened a PR for the related docs change (if not, why?) already addressed in #3919
  <link to docs PR>

## Testing Instructions

1. run metadata from this branch
2. verify  PATCH `/api/v2/deviceprofile/basicinfo` is functioning

(dependent on https://github.com/edgexfoundry/go-mod-core-contracts/pull/715)

## New Dependency Instructions (If applicable)
